run/hello-broken/README.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Run Broken Sample
+
+This sample presents broken code in need of troubleshooting. An alternate resource at /improved shows a more stable implementation with more informative errors and default values.
```

Comment:
- at `/improved`
- I'd make the second sentence a lot shorter and add a link
  - View [`improved`](https://www.youtube.com/watch?v=oHg5SJYRHA0) for a more stable implementation.

Reply:
That link just made my day.   🥇